docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,484 @@ Review the following information about the {kib} 8.3.0 release.
+The v1 and v2 job configurations for security_linux and security_windows are deprecated. For more information, check {kibana-pull}131166[#131166].
+
+*Impact* +
+The following security_linux and security_windows job configurations are updated to v3:
+
+* security_linux:
+
+** v3_linux_anomalous_network_activity
+** v3_linux_anomalous_network_port_activity_ecs
+** v3_linux_anomalous_process_all_hosts_ecs
+** v3_linux_anomalous_user_name_ecs
+** v3_linux_network_configuration_discovery
+** v3_linux_network_connection_discovery
+** v3_linux_rare_metadata_process
+** v3_linux_rare_metadata_user
+** v3_linux_rare_sudo_user
+** v3_linux_rare_user_compiler
+** v3_linux_system_information_discovery
+** v3_linux_system_process_discovery
+** v3_linux_system_user_discovery
+** v3_rare_process_by_host_linux_ecs
+
+* security_windows:
+
+** v3_rare_process_by_host_windows_ecs
+** v3_windows_anomalous_network_activity_ecs
+** v3_windows_anomalous_path_activity_ecs
+** v3_windows_anomalous_process_all_hosts_ecs
+** v3_windows_anomalous_process_creation
+** v3_windows_anomalous_script
+** v3_windows_anomalous_service
+** v3_windows_anomalous_user_name_ecs
+** v3_windows_rare_metadata_process
+** v3_windows_rare_metadata_user
+** v3_windows_rare_user_runas_event
+** v3_windows_rare_user_type10_remote_login
+====
+      
+[discrete]
+[[deprecation-130336]]
+.Updates the default legend size
+[%collapsible]
+====
+*Details* +
+In the *Lens* visualization editor, the *Auto* default for *Legend width* has been deprecated. For more information, check {kibana-pull}130336[#130336].
+
+*Impact* +
+When you create *Lens* visualization, the default for the *Legend width* is now *Medium*. 
+====
+      
+[discrete]
+[[deprecation-122075]]
+.Deprecates `xpack.data_enhanced.*`
+[%collapsible]
+====
+*Details* +
+In kibana.yml, the `xpack.data_enhanced.*` setting is deprecated. For more information, check {kibana-pull}122075[#122075].
+
+*Impact* +
+Use the `data.*` configuration parameters instead.
+====
+      
+[float]
+[[features-8.3.0]]
+=== Features
+
+{kib} 8.3.0 adds the following new and notable features.
+
+Alerting::
+* Adds circuit breaker for max number of actions by connector type {kibana-pull}128319[#128319]
+* Adds `bulkEdit` method to alerting rulesClient and internal _bulk_edit API, that allow bulk editing of rules {kibana-pull}126904[#126904]
+
+Cases::
+* Adds average time to close metric in Cases {kibana-pull}131909[#131909]
+* View all alerts attached to a case in the alerts table. The feature is experimental {kibana-pull}131883[#131883]
+* Adds severity field to Cases {kibana-pull}131626[#131626]
+* Adds the ability to delete comments in Cases {kibana-pull}130254[#130254]
+
+Dashboard::
+Enables the new controls by default {kibana-pull}131341[#131341]
+
+Discover::
+* Adds the ability to edit dataView, query, & filters {kibana-pull}131688[#131688]
+* Extend Elasticsearch query rule with search source based data fetching {kibana-pull}124534[#124534]
```

Comment:
This PRs were part of the initiative to enable threshold alerts in Discover, details here:

https://github.com/elastic/kibana/issues/117532

Should we create a high level issue to sum up the change? this is a significant change in Discover we should highlight

Reply:
@kertal are both #131688 and #124534 part of the efforts to enable threshold Alerts in Discover?

Reply:
Yes, exactly!

Reply:
works for me, thx!